README.rst

```diff
@@ -71,15 +71,15 @@ Example
 
         async def update_stats(self):
             """This function runs every 30 minutes to automatically update your server count"""
-           await self.bot.is_ready()
-            while not bot.is_closed:
```

Comment:
This is needed so that the loop will exit cleaning when the bot does. A while True loop leads to issues when the bot exits.